cpp/include/cuspatial/types.hpp

```diff
@@ -36,7 +36,7 @@ struct location_3d {
  * @tparam T the base type for the coordinates
  */
 template <typename T>
-struct coord_2d {
+struct vec_2d {
```

Comment:
I didn't mean you should rename this. This renaming is part of #477 which is about to merge.